js/shopping-list.js

```diff
@@ -0,0 +1,44 @@
+// let Item = require('../js/shopping-list-item.js');
+
+class List {
+  constructor() {
+    this.items = [];
+
+  }
+  addItem(item) {
+    if (item instanceof Item) {
+      this.items.push(item);
+    } else {
+      throw new Error;
```

Comment:
When throwing errors, it's best to invoke them and pass in a string that explains what kind of error this is. For example, this one would say something along the lines of `Error: Only Shopping List Items can be added to the Shopping List`